src/fitnesse/slim/SlimSlaveSocket.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2016 by six42@gmx.net, All rights reserved.
+
+package fitnesse.slim;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import util.FileUtil;
+import fitnesse.util.MockSocket;
+
+public class SlimSlaveSocket extends ServerSocket {
+  private static final Logger LOG = Logger.getLogger(SlimSlaveSocket.class
+      .getName());
+
+  private PrintStream stdout;
+  private PrintStream stderr;
+  private InputStream stdin;
+
+  public SlimSlaveSocket() throws IOException {
+
+    // preserve original streams
+    this.stdout = System.out;
+    this.stderr = System.err;
+    this.stdin = System.in;
+
+    // bind System.stdout/System.stderr to original stderr
+    System.setOut(new PrintStream(new LoggingOutputStream(this.stderr, "SOUT"),
```

Comment:
@amolenaar  
This redirects all messages send to stdout to stderr. So stdout is free. SLIM can send results without the SUT mixing in any log messages.
In line 58 in the "accept" method stdout is bound as communication channel for SLIM